packages/panzer/disc-fe/src/evaluators/Panzer_GatherSolution_BlockedTpetra_impl.hpp

```diff
@@ -389,6 +390,7 @@ evaluateFields(typename TRAITS::EvalData workset)
           const int rowLID = worksetLIDs(cell,fieldOffsets(basis));
 	       fieldValues(cell,basis).zero();
           fieldValues(cell,basis).val() = kokkosSolution(rowLID,0);
+          //std::cout << " C B VAL " << cell << " " << basis << " " << fieldValues(cell,basis) << " " << kokkosTangents(0)(cell,basis) << std::endl;
           for (int i_tangent=0; i_tangent<numTangents; ++i_tangent)
             fieldValues(cell,basis).fastAccessDx(i_tangent) = kokkosTangents(i_tangent)(cell,basis);
```

Comment:
@rppawlo This is what is seeming strange to me: `fieldValues` is populated here during evaluation if I print it out. But the PHX debug shows that the temperature field (tangent type) is zero before and after evaluation. Ultimately, the probe also see an all-zero field.